tools/ldb_cmd.cc

```diff
@@ -3770,6 +3807,48 @@ void DBQuerierCommand::DoCommand() {
           oss << "get " << key << " error: " << s.ToString();
         }
       }
+    } else if (cmd == COUNT_CMD) {
+      std::string start_key;
+      std::string end_key;
+      bool bad_option = false;
+      for (auto& option : option_map) {
```

Comment:
Seems to me this will always be empty since it's now contained in `parsed_params`.